bokeh/util/serialization.py

```diff
@@ -569,6 +570,18 @@ def decode_base64_dict(data):
         array = array.reshape(data['shape'])
     return array
 
+def encode_base64_string(str, value):
+    ''' Encode a ascii string using Base64.
+
+    Args:
+        str : a string to encode
+
+    Returns:
+        string
+
+    '''
+    b64_string = base64.b64encode(value.encode("ascii"))
```

Comment:
Why limit strings to ASCII? `utf-8` encoding seems to be a much better choice in my opinion.

Reply:
Surely utf-8 is better than ASCII, I will change it.

Reply:
Is this extra function even necessary? I would not be opposed to just calling 
```
base64.b64encode(value.encode("utf-8"))
``` 
directly in `Base64String` and avoiding adding new functions.

Reply:
I think this sounds better, I shall remove this and define base64 directly into `Base64String`.